src/pages/salesOfferPackages.tsx

```diff
@@ -0,0 +1,238 @@
+import React, { ReactElement } from 'react';
+import { BaseLayout } from '../layout/Layout';
+import { CustomAppProps, ErrorInfo, NextPageContextWithSession } from '../interfaces';
+import CsrfForm from '../components/CsrfForm';
+import ErrorSummary from '../components/ErrorSummary';
+import { getSessionAttribute } from '../utils/sessions';
+import { SALES_OFFER_PACKAGES_ATTRIBUTE } from '../constants';
+import FormElementWrapper from '../components/FormElementWrapper';
+
+const title = 'Sales Offer Packages - Fares Data Build Tool';
+const description = 'Sales Offer Packages page for the Fares Data Build Tool';
+
+export const ticketsPurchasedList = {
+    id: 'ticketsPurchasedFrom',
+    method: [
+        'OnBoard',
+        'Online',
+        'Online Account',
+        'Telephone',
+        'Electronic Pass',
+        'Mobile Device',
+        'Agency',
+        'Tour Operator',
+    ],
+};
+
+export const ticketPaymentMethodsList = {
+    id: 'ticketPayments',
+    paymentMethod: [
+        'Cash',
+        'Debit/Credit card',
+        'Contactless card',
+        'Mobile Payment',
+        'SMS',
+        'Cheque',
+        'Direct transfer',
+        'Standing Order',
+        'Warrant',
+        'Gift Voucher',
+    ],
+};
+
+export const ticketFormatsList = {
+    id: 'ticketFormats',
+    ticketFormats: ['Paper Ticket', 'Mobile App', 'Smart Card'],
+};
+
+interface SalesOfferPackage {
+    selected: string[];
+}
+
+export interface SalesOfferPackagesProps {
+    ticketsPurchasedFrom: SalesOfferPackage;
+    ticketPayments: SalesOfferPackage;
+    ticketFormats: SalesOfferPackage;
+    errors: ErrorInfo[];
+}
+
+export const SalesOfferPackagesInfo = () => (
```

Comment:
resuable so can be utilised by the page after. Keeping it isolated as it only refers to offer packages